Algorithmia/algorithm.py

```diff
@@ -29,6 +37,97 @@ def set_options(self, timeout=300, stdout=False, output=OutputType.default, **qu
         self.query_parameters.update(query_parameters)
         return self
 
+    # Create a new algorithm
+    def create(self, details={}, settings={}, version_info={}):
+        detailsObj = Details(**details)
+        settingsObj = SettingsMandatory(**settings)
+        createRequestVersionInfoObj = CreateRequestVersionInfo(**version_info)
+        create_parameters = {"name": self.algoname, "details": detailsObj, "settings": settingsObj, "version_info": createRequestVersionInfoObj}
```

Comment:
Where does it get the username from? How does this work if I'm creating an org-owned algo?

Reply:
Nvm, now I see it below.